civis/cli/__main__.py

```diff
@@ -162,10 +162,12 @@ def invoke(method, path, op, *args, **kwargs):
     with open_session(get_api_key(), user_agent=CLI_USER_AGENT) as sess:
         response = sess.request(**request_info)
 
-    # Print the response to stderr if there was an error.
+    # Print the response to stderr and set exit code to 1 if there was an error
     output_file = sys.stdout
+    exit_code = 0
     if response.status_code != 200:
```

Comment:
hmmm... i think we might need to change this to capture any 2xx statuses rather than just 200.  i dunno if we use 201 errors anywhere, for example, but a 201 response for object creation is reasonable. https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201